From 1c3d2e79bb8119a734cfddb9d34e9dfd3839da14 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 11 Nov 2019 18:54:40 +0200
Subject: [PATCH] [SW-6379] goya2: Add -Werror to main Makefile

We do not want to allow any warning as coding convention

Change-Id: Ie60fb2660d53116e88a6a60a754c75edfc237c4b
---

diff --git a/Makefile b/Makefile
index fdbdcef..b166293 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@
 OBJCOPY = $(CROSS_COMPILE)objcopy
 
 LIBDIR = sdk/lib
-ARCBOOT_CFLAGS = -DSB_RSA_STD -DSB_FAST_MODE -DELP_SOURCE -Wall -W -O3
+ARCBOOT_CFLAGS = -DSB_RSA_STD -DSB_FAST_MODE -DELP_SOURCE -Wall -W -Werror -O3
 ARCBOOT_CFLAGS += -Isdk/src
 ARCBOOT_CFLAGS += -Isdk/src/include
 ARCBOOT_CFLAGS += -Isdk/elpsoft/src/headers
